### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mohitgusain**
Pleases review this contributing.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+        
+        $ git push --set-upstream origin branch name 
+  
+5. It's your responsibility to **self review and self test** your commits because it may conflict with base and cause errors.
+
+6. Always  done your pull request(PR) to the base  from your origin and for this repository base is **coloredcow/employee-portal**.
+
+7. After pull request write a  **description about your issue and what you have done in the code.**
+
+ 8.Atlast please leave a *feedback* and request for *review* the code.
```
Comment (pankaj-ag) [inline CONTRIBUTING.md]: Formatting.

Comment (pankaj-ag) [inline CONTRIBUTING.md]: It looks good however can we add some pointers which will help other contributors.

- We follow [PSR-2 coding standards ](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)
- Limited use of jquery. 
- Different casing for different HTML / javascript elements  classes and IDs
- Style CI.
- Unit Testing for the feature.

Comment (mohitgusain) [reply CONTRIBUTING.md]: Mean above these  points should  also mention on contributing.md ??

Comment (mohitgusain) [reply CONTRIBUTING.md]: @pankaj-ag  please tell me should i add above all those points which are above mentioned by you.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,46 @@
 # Contributing
 
 When contributing to this repository, please first discuss the change you wish to make via an issue before making a change.
+
+1.Before contributing keep this thing in your mind that sychronize your develop from base.
```
Comment (pankaj-ag) [inline CONTRIBUTING.md]: You mean synchronize right ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,46 @@
...
+2.For every issue or contributing in repository create a new branch  for this.
+
+        $ git branch branch name
+        -branch name should be something like this  < 221_create_new_column> ,221 is issue no.
+
+3.Try to frequently commit the changes.
+        
+        $ git commit -m <"appropriate message">
+     
+4.After commit push your commits to the the origin.
```
Comment (pankaj-ag) [inline CONTRIBUTING.md]: the the?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,3 +1,46 @@
...
+
+8.Atlast please leave a *feedback* and request for *review* the code.
+
+
+### Some key points
+1.Laravel follows the [PSR-2](https://www.php-fig.org/psr/psr-2/) coding standard and the [PSR-4](https://www.php-fig.org/psr/psr-4/) autoloading standard.
+
+2.[StyleCI](https://styleci.io/) provides checks for both your open and closed source repositories. By analyzing every push or pull request made on your codebase, StyleCI ensures that your code is always written against the standards you want it to be.
+
+3.Unit testing for feature , for more imformation [Testing](https://laravel.com/docs/5.6/testing)
```
Comment (pankaj-ag) [inline CONTRIBUTING.md]: imformation ?

**Comment by mohitgusain** [comment]
please review it again @pankaj-ag .

**Pull request merged**